### ISSUE TITLE
Carbon v9 pagination show s accessibility issues with DAP

### ISSUE THREAD

**Issue by SowmyaRaviKrishnan** [carbon-design-system/carbon#4656]
Carbon v9 pagination show s accessibility issues with DAP
1.INPUT LABEL references must be valid

## Environment

> Operating system - RHEL

> Browser - chrome

> Assistive technology used to verify - IBM DAP

## Detailed description

> What version of the Carbon Design System are you using? - v9

> What WCAG 2.1 checkpoint does the issue violate?
IBM 1.3.1 Info and Relationships

**Comment by SowmyaRaviKrishnan** [comment]
@joshblack @amuqeetkhan

**Comment by abbeyhrt** [comment]
Hi @SowmyaRaviKrishnan, thanks for bringing this to our attention! We will not be backporting WCAG 2.1 fixes to v9 but we always accept community contributions for any v9 issues! If you would like this to be addressed, feel free to open up a PR and we’ll review it! 👍

**Comment by amuqeetkhan** [comment]
Hello Everyone, This is not a v9 problem. We have tried this on v10 actually as per our package.json
```
"dependencies": {
    "@carbon/colors": "^10.2.0",
    "@carbon/grid": "^10.3.0",
    "@carbon/icons": "^10.3.0",
    "@carbon/icons-react": "^10.3.0",
    "@carbon/layout": "^10.2.0",
    "@carbon/type": "^10.2.0",
    "carbon-components": "^10.3.2",
    "carbon-components-react": "^7.3.2",
    "carbon-icons": "^7.0.7",
```
So we should consider this a v10 issue. @joshblack @abbeyhrt
